ANMN/burst_averaged_product/burst_average.py

```diff
@@ -148,12 +148,24 @@ def list_var_to_average(netcdf_file_obj):
     var_list = netcdf_file_obj.variables.keys()
     var_list = [x for x in var_list if not x.endswith('_quality_control')]
 
-    for var_to_remove in ['TIME', 'LATITUDE', 'LONGITUDE', 'NOMINAL_DEPTH', 'VOLT', 'TIMESERIES', 'SSPD', 'CNDC']:
-        if var_to_remove in var_list:
-            var_list.remove(var_to_remove)
+    var_to_remove = []
+    for varname in var_list:
+        if 'TIME' not in netcdf_file_obj.variables[varname].dimensions:
+            var_to_remove.append(varname)
+
```

Comment:
This is not what I suggested to do:
> We want to keep everything as it is, except:
-we get rid of any '_quality_control' variable.
-we duplicate any variable PARAM that is a function of TIME so that we have: PARAM, PARAM_min, PARAM_max, PARAM_sdv and PARAM_num_obs. These variables will automatically inherit the data and attributes from PARAM. Apply mean or median, min, max, standard_dev and count to the data of each of them respectively (flags are taken into account to only do this on "good" data) and update the relevant attributes accordingly, add cell_methods, etc...
-TIME is treated slightly differently since it doesn't have a TIME_min, TIME_max, etc...
-for global attributes, most of them will still be relevant, some will need to be deleted, some updated.

So any variable that is not a function of TIME is kept.